src/styles/bloks/card.scss

```diff
@@ -0,0 +1,148 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+
+  min-width: none;
+  max-width: 1024px;
+  margin: 0 auto;
+  padding-top: 50px;
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  @for $stars from 1 through 4 {
+    &--#{$stars} &__star:nth-child(-n + #{$stars}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: 50px;
+  margin-bottom: 48px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &_img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    margin-left: 3px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+  }
+
+  &__link {
+    height: 40px;
+  }
```

Comment:
do not hardcode height, use padding instead

Reply:
I cannot using padding because past I was usind padding I break styles from Figma